site/content/contribute/server/developer-setup/ubuntu.md

```diff
@@ -3,6 +3,7 @@
     ```sh
     curl -fsSL https://get.docker.com -o get-docker.sh
     sudo sh get-docker.sh
+    sudo usermod -aG docker {username}
```

Comment:
Can you replace `{username}` with `$(whoami)` to make it even easier to copy and paste?

Reply:
Yes, Sure. That is a better solution.